modules/extensions/apps/portal/store/carbon.super/fs/gadget/task_instance_count_vs_task_id/index.xml

```diff
@@ -87,7 +92,7 @@
                     <button class="btn btn-primary" onclick="drawGraph()">Apply Filter</button>
                 </div>
             </div>
-            <div id="chartA"></div>
+            <div id="chartA" style="width:100%; height:100%; padding-top:20px"></div>
```

Comment:
Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7